hnn_core/network_builder.py

```diff
@@ -81,6 +82,20 @@ def simulation_time():
     neuron_net.aggregate_data()
     _PC.allreduce(neuron_net.dipoles['L5_pyramidal'], 1)
     _PC.allreduce(neuron_net.dipoles['L2_pyramidal'], 1)
+    for elec in neuron_net._lfp:
+        if len(elec._lfp_v) == len(times) - 1:
+            # Last value repeated, CVode doesn't update the membrane currents
+            # for the last time step.
+            elec.calc_potential_callback()
+        elif len(elec._lfp_v) == len(times):
+            pass
+        else:
+            raise RuntimeError(
+                f'Length of LFP potential vector ({len(elec._lfp_v)}) does not'
+                f' match number of time points in simulation ({len(times)})')
+        _PC.allreduce(elec._lfp_v, 1)
+    for nrn_arr in neuron_net._lfp_array.values():
+        _PC.allreduce(nrn_arr._lfp_v, 1)
```

Comment:
shouldn't this be in `neuron_net.aggregate_data`  ?

Reply:
It doesn't look to me like `aggregate_data` is doing this.

Reply:
`aggregate_data` is a specialty method to deal with the problem of iterating over each individual cell and collecting their associated voltages/currents. Perhaps it should be named `aggregate_cell_data` in a MAINT PR?